test/integration/config/prow/cluster/deck_tenant_deployment.yaml

```diff
@@ -55,7 +55,6 @@ spec:
         - --github-token-path=/etc/github/oauth
         - --github-endpoint=http://fakeghserver
         - --github-oauth-config-file=/etc/githuboauth/secret
-        - --cookie-secret=/etc/cookie/secret
```

Comment:
why do you need to remove the cookie-secret flag here? (this probably fails the integration test)

Reply:
I was trying it to see if it would make the test pass. It seems that it doesn't help. I am picking this back up today, and will try to figure out what is making it fail.